extensions/grpc/protoc/pom.xml

```diff
@@ -25,4 +25,25 @@
             </exclusions>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
What's the reason for building a fat jar?

Reply:
For protoc, I need to construct a single executable for this plugin. I can't even do `java -jar ...`, so I'm wrapping it in a script: https://github.com/michalszynkiewicz/quarkus/blob/code-gen-grpc/extensions/grpc/deployment/src/main/java/io/quarkus/grpc/deployment/GrpcCodeGen.java#L166

Having the plugin as a fat jar simplifies construction of the script (no need to gather classpath). @aloubyansky suggested this approach

Reply:
The second commit still shades the jar but deploys with `shaded` classifier. Otherwise devmode tests for the `grpc/deployment` module had problems finding the artifact.